components/ai_chat/content/browser/ai_chat_tab_helper.h

```diff
@@ -70,11 +70,16 @@ class AIChatTabHelper : public content::WebContentsObserver,
   // Delegate to extract print preview content
   class PrintPreviewExtractionDelegate {
    public:
-    using ExtractCallback = base::OnceCallback<void(std::string)>;
+    using ExtractCallback =
+        base::OnceCallback<void(base::expected<std::string, std::string>)>;
+    using CapturePdfCallback = base::OnceCallback<void(
+        base::expected<std::vector<std::vector<uint8_t>>, std::string>)>;
```

Comment:
nit: Seems like a good place to state out what exactly std::string or std::vector<std::vector<uint8_t>> refers to in the result. Though it probably can be inferred by looking at Extract and CapturePdf's comment, so I'm fine either way.

Reply:
added some comments for both